clang/test/CodeGen/fp-accuracy.c

```diff
@@ -450,7 +450,7 @@ void f1(float a, float b) {
 // CHECK-SPIR: call double @llvm.fpbuiltin.tan.f64(double {{.*}}) #[[ATTR_SYCL2]]
 // CHECK-SPIR: call double @llvm.fpbuiltin.log10.f64(double {{.*}}) #[[ATTR_SYCL5]]
 // CHECK-SPIR: call void @llvm.fpbuiltin.sincos.f32(float {{.*}}, ptr {{.*}}, ptr {{.*}}) #[[ATTR_SYCL1]]
-// CHECK-SPIR: call spir_func float @tanf(float noundef {{.*}})
+// CHECK-SPIR: call float @llvm.fpbuiltin.tan.f32(float {{.*}})  #[[ATTR_SYCL2]]
```

Comment:
The other attributes seem self-explanatory, but if you could add a comment for what the 'fpbuiltin-max-error"="1.0"' and other values mean, that would be helpful to those looking at this test later.